### PR TITLE
PR: Lattice documentation changes.

### LINKED ISSUE #150: Fix documentation to mention that lattice attributes are always 3D.
#80 was addressed with PR #147 but the documentation (e.g. docstrings, uml) still needs to be done

### PR DESCRIPTION
**Pull request by tuopuu**
This PR fixes lattice related uml diagrams so that they correspond to current implementation. This PR fixes #150.

Changes: 
* containers.png
* h5lattice.png
* lattice.txt
* h5lattice.txt

### CODE REVIEW

**Comment by tuopuu** [comment]
@roigcarlo please, take a look.

File: doc/source/uml/lattice.txt [hunk 1]
```diff
@@ -2,13 +2,13 @@
 abstract class ABCLattice {
   name: str
   type: str
-  base_vect: tuple of double[2..3]
-  size: tuple of double[2..3]
-  origin: tuple of double[2..3]
+  base_vect: float[3]
+  size: tuple of int[3]
```
Comment (itziakos) [inline doc/source/uml/lattice.txt]: should this be int[3]?

**Comment by itziakos** [comment]
@tuopuu, just one comment, apart from that :+1:

File: doc/source/uml/lattice.txt (hunk 1 above)
Comment (tuopuu) [inline doc/source/uml/lattice.txt]: Thanks @itziakos. Current implementation (or docstrings) of ABCLattice and Lattice seem to have the size attribute as tuple of int[3], which is an unneccessary requirement. I'll change both of them to int[3], and the uml diagrams as well.

**Pull request merged**